README.md

```diff
@@ -200,6 +200,51 @@ There are also a couple of how-to guides:
 * [Natural sort Table](docs/tableWithNaturalSort.md)
 * [Sorting a Table by multiple columns](docs/multiColumnSortTable.md)
 
+Testing
+---------------
+As this library does not come pre-compiled, it requires to be transformed by your loader, otherwise you may get errors such as:
```

Comment:
This depends on which version you're using. for example, react-virtualized also ships cjs builds which in this case would not break

Reply:
*common js

Reply:
My issue was due to Create React App being setup by default to not transpile anything in node modules.

I've seen some people copy the code of default row renderer into their code to get around this. But for someone new to the ecosystem it wasn't obvious what and how to fix. I am happy to change the wording to anything, but definitely think it is worth mention to help someone else.